src/components/Wizard/CreateVmWizard/StorageTab.js

```diff
@@ -335,6 +393,18 @@ class StorageTab extends React.Component {
             .filter(pvc => !this.state.rows.some(row => row.name === pvc)),
           initialValue: '--- Select Storage ---',
         },
+        {
```

Comment:
For the sake of maintainability, please change `renderConfigs` to function or map.

Reply:
Follow up https://github.com/kubevirt/web-ui-components/issues/132